lib/digital_public_works/projects/project.ex

```diff
@@ -20,7 +21,7 @@ defmodule DigitalPublicWorks.Projects.Project do
   @doc false
   def changeset(project, attrs) do
     project
-    |> cast(attrs, [:title, :body])
+    |> cast(attrs, [:title, :body, :is_public])
```

Comment:
Perhaps we could have a second changeset function called admin_changeset (like the User.create_changeset) that casts is_public.

Then we could pass the current user to Projects.update_project (like create_project) and pattern match on is_admin to determine which changeset to use.

We could modify Projects.create_project to follow a similar pattern.